src/esp/sensor/CameraSensor.cpp

```diff
@@ -66,50 +71,9 @@ void CameraSensor::recomputeBaseProjectionMatrix() {
   recomputeProjectionMatrix();
 }  // CameraSensor::recomputeNearPlaneSize
 
-CameraSensor& CameraSensor::setProjectionMatrix(
-    gfx::RenderCamera& targetCamera) {
-  targetCamera.setProjectionMatrix(width_, height_, projectionMatrix_);
-  return *this;
-}
-
-CameraSensor& CameraSensor::setTransformationMatrix(
-    gfx::RenderCamera& targetCamera) {
-  CORRADE_ASSERT(!scene::SceneGraph::isRootNode(targetCamera.node()),
-                 "CameraSensor::setTransformationMatrix: target camera cannot "
-                 "be on the root node of the scene graph",
-                 *this);
-  Magnum::Matrix4 absTransform = this->node().absoluteTransformation();
-  Magnum::Matrix3 rotation = absTransform.rotationScaling();
-  Magnum::Math::Algorithms::gramSchmidtOrthonormalizeInPlace(rotation);
-
-  VLOG(2) << "||R - GS(R)|| = "
-          << Eigen::Map<mat3f>((rotation - absTransform.rotationShear()).data())
-                 .norm();
-
-  auto relativeTransform =
-      Magnum::Matrix4::from(rotation, absTransform.translation()) *
-      Magnum::Matrix4::scaling(absTransform.scaling());
-
-  // set the transformation to the camera
-  // so that the camera has the correct modelview matrix for rendering;
-  // to do it,
-  // obtain the *absolute* transformation from the sensor node,
-  // apply it as the *relative* transformation between the camera and
-  // its parent
-  auto camParent = targetCamera.node().parent();
-  // if camera's parent is the root node, skip it!
-  if (!scene::SceneGraph::isRootNode(
-          *static_cast<scene::SceneNode*>(camParent))) {
-    relativeTransform =
-        camParent->absoluteTransformation().inverted() * relativeTransform;
-  }
-  targetCamera.node().setTransformation(relativeTransform);
-  return *this;
-}
-
-CameraSensor& CameraSensor::setViewport(gfx::RenderCamera& targetCamera) {
-  targetCamera.setViewport(this->framebufferSize());
-  return *this;
+void CameraSensor::recomputeProjectionMatrix() {
+  projectionMatrix_ = zoomMatrix_ * baseProjMatrix_;
+  // renderCamera_->setProjectionMatrix(width_, height_, projectionMatrix_);
```

Comment:
why is it commented out?

Reply:
Sorry, was testing something out and forgot to uncomment! Will fix